lms/djangoapps/grades/tests/test_scores.py

```diff
@@ -47,20 +51,20 @@ class TestGetScore(TestCase):
     display_name = 'test_name'
     location = 'test_location'
 
-    SubmissionValue = namedtuple('SubmissionValue', 'exists, weighted_earned, weighted_possible')
-    CSMValue = namedtuple('CSMValue', 'exists, raw_earned, raw_possible')
+    SubmissionValue = namedtuple('SubmissionValue', 'exists, points_earned, points_possible, created_at')
```

Comment:
I understand that the actual names returned by the Submissions API isn't renamed here.  My concern is in this file itself.  To reiterate, the Grades team spent multiple times fixing issues related to confusion between `raw` and `weighted` - and we therefore updated various places in the code to explicitly make this distinction.  This change is regressing back in the other direction - renaming to generic "points"  instead of being explicit about whether those points represent "weighted" versus "raw".